doc/install/install-openshift.md

```diff
@@ -27,9 +27,27 @@ Before you can use Kuadrant, you must install Gateway API v1 as follows:
 kubectl apply -f https://github.com/kubernetes-sigs/gateway-api/releases/download/v1.0.0/standard-install.yaml
 ```
 
-### Step 3 - Install and configure Istio with the Sail Operator
+### Step 3 - Install cert-manager
 
-Kuadrant integrates with Istio as a Gateway API provider. You can set up an Istio-based Gateway API provider by using the Sail Operator. 
+Before you can use Kuadrant, you must install cert-manager.
+
+Install one of the different flavours of the Cert-Manager.
+
+#### Install community version of the cert-manager
```

Comment:
do we want to specify a version or min version?

Reply:
We did not specify version or require min version until now. 

Is there really a min version of cert-manager that kuadrant supports? @mikenairn ?

Reply:
Actually, we did when the dependency was based on the operator, not on the API: 

From https://github.com/Kuadrant/kuadrant-operator/pull/680/files#diff-14e7feda952796eb377d3d14ab302dccca03b212ce86cfddea587f95a8351cebL16-L17

```yaml
- type: olm.package
    value:
      packageName: cert-manager
      version: "1.14.2"
```

Adding that.

Reply:
The version of cert manager we are currently testing with, at least from this repo, is "1.12.1" https://github.com/Kuadrant/kuadrant-operator/blob/main/config/dependencies/cert-manager/kustomization.yaml#L2. Not sure what min version we support really, but I'd probably just say that version.

Reply:
done